test/test_nn.py

```diff
@@ -13368,6 +13368,58 @@ def helper(n, c, h, w, ks):
         helper(2, 8, 4, 4, ks=2)
         helper(None, 3, 50, 50, ks=5)
 
+    @onlyCPU
+    @dtypes(torch.float, torch.double)
+    def test_adaptive_pooling_max_nhwc(self, device, dtype):
+        input = torch.randint(1, 10, (4, 8, 8, 8), dtype=dtype).to(device)
```

Comment:
please test input with non-square width and height, to avoid accidental bugs where sizes are swapped in kernels

Reply:
add non square width and height as well as output_height and output_width